docs/FAQ.rst

```diff
@@ -0,0 +1,286 @@
+#############################################
+A: Yes, DALI supports data-parallel and distributed data-parallel strategies (you can read more
+about these strategies `here <https://pytorch.org/tutorials/intermediate/ddp_tutorial.html#comparison-between-dataparallel-and-distributeddataparallel>`_).
+Its shards data into non-overlapping pieces using the number of shards (world size) and shard id (global rank), and
+uses device id to identify the GPU used in the particular node (local rank).
+
+More details can be also found it `this documentation section <../advanced_topics_sharding.html>`_
+
+Q: How to report an issue/RFE or get help with DALI usage?
+##########################################################
+A: DALI is an open-source project hosted on GitHub, you can ask questions and report issues
+using `this link <https://github.com/NVIDIA/DALI/issues>`_ directly.
+
+Q: Can DALI accelerate the loading of the data, not just processing?
+####################################################################
+A: DALI mostly focuses on processing acceleration, as in most cases the input data is compressed
+(audio, video, or images) and the input data is relatively small compared to the raw decoded output.
+Still, there are cases, where data is not compressed and loading it directly to the GPU is feasible.
+To support that case DALI can use `GPUDirect Storage <https://developer.nvidia.com/gpudirect-storage>`_
+inside Numpy GPU reader to bypass CPU and load the data directly to the GPU.
+
+Q: How can I obtain DALI?
+#######################################################
+A: DALI is available as a prebuilt python wheel binary -
+`see to learn how to install it <https://docs.nvidia.com/deeplearning/dali/user-guide/docs/installation.html>`_
+or as `a source code <https://github.com/NVIDIA/DALI>`_ that can be built on your own.
+
+Q: Which OS does DALI support?
+##############################
+A: DALI does support all major Linux distributions and indirectly Windows through
+`WSL <https://docs.nvidia.com/cuda/wsl-user-guide/index.html>`_. Regrettably, MacOS
+is not unsupported.
+
+Q: Where can I find the list of operations that DALI supports?
+##############################################################
+A: You can find a comprehensive list of operators available `here <../supported_ops.html>`_.
+
+Q: Can I send a request to the Triton server with a batch of samples of different shapes (like files with different lengths)?
+#############################################################################################################################
+A: Even though DALI supports such data, Triton doesn't accept non-uniform batches. You can approach
+it in two ways. The first is to pad the samples to make a uniform batch. The second is to split
+the request so you send together only samples with the same shape - in some cases, they can be
+later internally merged in the Triton server thanks to the dynamic batching feature.
```

Comment:
This is actually not true, Triton may accept the ragged batch. I suggest modifying the answer:
```suggestion
A: Batch processing is one of main DALI assumptions. On the other hand, Triton Inference Server supports a uniform batch by default. However, by enabling a `ragged batching <https://github.com/triton-inference-server/server/blob/main/docs/ragged_batching.md>`_ you can send non-uniform batches and process them successfully. `Here <https://github.com/triton-inference-server/dali_backend/blob/7d51c7299dd66964097f839501e18f3b579cc306/qa/L0_DALI_GPU_ensemble/client.py#L31>`_ you can find an example of using ragged batching feature with DALI Backend.
```

In the Triton's ragged batch documentation there's no info about DALI, I'll add it there.